xml/System.Net/WebHeaderCollection.xml

```diff
@@ -290,6 +290,11 @@
  [!code-csharp[WebHeaderCollection_Add#1](~/samples/snippets/csharp/VS_Snippets_Remoting/WebHeaderCollection_Add/CS/webheadercollection_add.cs#1)]
  [!code-vb[WebHeaderCollection_Add#1](~/samples/snippets/visualbasic/VS_Snippets_Remoting/WebHeaderCollection_Add/VB/webheadercollection_add.vb#1)]  
   
+> [!NOTE]
+> The length of `value`, i.e. the string after the colon (:), is validated only in .NET Framework and .NET Core.
+> - On all applicable .NET Framework versions: <xref:System.Net.WebHeaderCollection> instance that is returned by the <xref:System.Net.HttpListenerResponse.Headers%2A> property will throw <xref:System.ArgumentOutOfRangeException> if the length of `value` is greater than 65535. All other <xref:System.Net.WebHeaderCollection> instances will accept `value` of any length.
```

Comment:
```suggestion
> - On all applicable .NET Framework versions: A <xref:System.Net.WebHeaderCollection> instance that's returned by the <xref:System.Net.HttpListenerResponse.Headers%2A> property will throw an <xref:System.ArgumentOutOfRangeException> if the length of `value` is greater than 65535. All other <xref:System.Net.WebHeaderCollection> instances accept a `value` of any length.
```

Can you make this change throughout the file?

Reply:
Fixed.